src/utils/error.js

```diff
@@ -4,14 +4,12 @@
  */
 class ErrorHandler extends Error {
   /**
-     * Create error handler.
-     * @param {string} message - The message to be passed for the error.
-     * @param {number} statusCode - The status code of the error.
-     */
-  constructor(message, statusCode = 500) {
-    super();
-    this.statusCode = statusCode;
-    this.message = message;
+   * use to wrap around controller to avoid try catch blocks
+   * @param {function} fn
+   * @returns {function} next
+   */
+  static catchErrors(fn) {
```

Comment:
I think we should avoid classes and static methods and defer to objects:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Details_of_the_Object_Model